docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -216,16 +218,17 @@ under ``localhost:7990``.
 
 #. Download the
    `bamboo-server-notification-plugin <https://github.com/ls1intum/bamboo-server-notification-plugin/releases>`__
-   and add it to bamboo. Go to Bamboo → Manage apps → Upload app → select
+   and add it to bamboo. Go to `Bamboo → Manage apps <http://localhost:8085/plugins/servlet/upm>`__ → Upload app → select
    the downloaded .jar file → Upload
 
-#. Authorize the Bamboo agent. Bamboo Administration → Agents → Remote agents → Agent authentication
+#. Authorize the Bamboo agent. `Bamboo Administration → Agents <http://localhost:8085/admin/agent/configureAgents!doDefault.action>`__
+   → Remote agents → Agent authentication
```

Comment:
These links are not working for me when using the provided artifact. Why do you use localhost here?

Reply:
Those are the default links for the atlassian suite when starting Artemis locally with docker right?

I think those links make sense in the context of the entire setup

Reply:
I am not familiar with using the artifacts, are the other parts of https://artemis-platform.readthedocs.io/en/latest/dev/setup.html#bamboo-bitbucket-and-jira-setup relevant for your setup?
Then the localhost links are rather confusing and I will remove them!

Reply:
Ah yes, I thought this should have linked to something in the documentation itself 🤦
The links also work for me, when I have started the Atlassian suite